src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -461,6 +471,11 @@ public virtual void Abort()
                                     await WriteHandshakeResponseAsync(HandshakeResponseMessage.Empty);
                                     return true;
                                 }
+                                else if (overLength)
+                                {
+                                    Log.MessageTooBig(_logger, maxMessageSize.Value);
```

Comment:
Feels like we should write a response. Won't the client hang?

Reply:
Ah I guess this ends the connection. We should write something in a response though.

Reply:
Agreed